### ISSUE TITLE
Implement a pre-trained deep-learning model (such as BERT, XML, etc.) 

### ISSUE THREAD

**Issue by mattesko** [mattesko/COMP550-Project#12]
If there's enough time and the minimum requirements of the project are met (all MVP tags are complete), then implementing and evaluating a pre-trained deep-learning model could help us achieve higher classification accuracy.

Ideas:
- Use one of PyTorch Transformers package's pre-trained models
